src/@next/components/organisms/SelectSidebar/SelectSidebar.tsx

```diff
@@ -0,0 +1,69 @@
+import { Trans } from "@lingui/react";
+import React from "react";
+
+import { ButtonLink } from "@components/atoms";
+import { CardHeader, OverlayItem } from "@components/molecules";
+import { useHandlerWhenClickedOutside } from "@hooks";
+
+import { Overlay } from "../";
+import * as S from "./styles";
+import { IProps } from "./types";
+
+export const SelectSidebar: React.FC<IProps> = ({
+  title,
+  options = [],
+  disabled = [],
```

Comment:
consider renaming `disabled` & `selected` to `disabledOptions`  and `selectedOptions` to be more specific. Otherwise at first it suggests the entire Sidebar might be disabled.